### DISCUSSION TITLE
Is the --webserver chat paid now?

### DISCUSSION THREAD

**Discussion by junaid33** [TabbyML/tabby#1690] [Q&A] [answered]
I came back to test Tabby after a couple of months today. When I run the cli and try to access the chat at localhost:8080/dashboard, it redirects me to a login screen? What is the user and password? Or is the webserver a EE feature now? If that is the case, it should be relayed to the users in the release notes.

**Comment by junaid33** [comment, marked as answer]
Nevermind, I had created a user and just forgot the password. For anyone else facing this issue, go to .tabby/ee/db.sqlite and delete the user. The webserver will ask you to create a new user since no users exist in the DB.

**Comment by wsxiaoys** [comment]
Right - right now there's no way to reset admin password (root user) unless smtp is configured.
